frontend_app_share_folder_advanced/src/container/ShareFolderAdvanced.jsx

```diff
@@ -249,6 +249,7 @@ class ShareFolderAdvanced extends React.Component {
           componentTitle={<div>{title}</div>}
           userRoleIdInWorkspace={state.loggedUser.userRoleIdInWorkspace}
           onClickCloseBtn={this.handleClickBtnCloseApp}
+          hideChangeTitleButton
```

Comment:
a good practice in general development is to use "positive" variable names.
It's more intuitive to read `if (showHeaderBar) { displayHeaderBar() }` than `if (!hideHeaderBar) { displayHeaderBar }`.
I'd like you to rename this props and it's usage.
Also, give it a default props to `true` to not break other potential usage and add it to the propTypes